### DISCUSSION TITLE
GDAP Invite Wizard - Fails to generate Invite

### DISCUSSION THREAD

**Discussion by jleroxGWNTC** [KelvinTegelaar/CIPP#2268] [General]
We're encountering an issue with the GDAP Invite Wizard
We have followed the documentation, we have all the correct Partner roles assigned the CIPP-Service Account and we cannot generate 
GDAP invites from CIPP. 

![image](https://github.com/KelvinTegelaar/CIPP/assets/146362232/46d06c21-1f19-4c11-a0b1-49ca47a68541)

**Comment by rvdwegen** [comment]
Check CIPP logs, should show more info

**Reply by jleroxGWNTC** [reply]
When I check the logs, I see this error
Error creating GDAP relationship: There were issue(s) with the input. See details for more information.
followed by "Created GDAP Invite -"

![image](https://github.com/KelvinTegelaar/CIPP/assets/146362232/356c30aa-0956-42e5-b6cf-e42b8d3ca348)

**Comment by KelvinTegelaar** [comment]
This is a support question, support is only for sponsors to our helpdesk email address :)

**Reply by jleroxGWNTC** [reply]
Sent :)
